Mechanics/Module.cs

```diff
@@ -6,9 +6,38 @@ namespace SpaceOpera.Mechanics
 {
     class Module
     {
-        string name;
-        Size size;
-        Dictionary<SkillName, int> skillModification;
+        public Module(string name, Size size, Dictionary<SkillName, int> skillModification, int cargoModificatio, int hpModification, int shieldModification, Weapon weapon)
+        {
+            Name = name;
```

Comment:
Istnieje w C# jakaś forma this.name?

Reply:
Istnieje, ale w tym kontekście lepiej zostawić tak jak jest :D